backend_new/prisma/schema.prisma

```diff
@@ -0,0 +1,1025 @@
+  neighborhood                          String?
+  petPolicy                             String?                       @map("pet_policy")
+  smokingPolicy                         String?                       @map("smoking_policy")
+  unitsAvailable                        Int?                          @map("units_available")
+  unitAmenities                         String?                       @map("unit_amenities")
+  servicesOffered                       String?                       @map("services_offered")
+  yearBuilt                             Int?                          @map("year_built")
+  applicationDueDate                    DateTime?                     @map("application_due_date") @db.Timestamptz(6)
+  applicationOpenDate                   DateTime?                     @map("application_open_date") @db.Timestamptz(6)
+  applicationFee                        String?                       @map("application_fee")
+  applicationOrganization               String?                       @map("application_organization")
+  applicationPickUpAddressOfficeHours   String?                       @map("application_pick_up_address_office_hours")
+  applicationPickUpAddressType          ApplicationAddressTypeEnum?   @map("application_pick_up_address_type")
+  applicationDropOffDddressOffice_hours String?                       @map("application_drop_off_address_office_hours")
+  applicationDropOffAddress_type        ApplicationAddressTypeEnum?   @map("application_drop_off_address_type")
+  applicationMailingAddressType         ApplicationAddressTypeEnum?   @map("application_mailing_address_type")
+  buildingSelectionCriteria             String?                       @map("building_selection_criteria")
+  costsNotIncluded                      String?                       @map("costs_not_included")
+  creditHistory                         String?                       @map("credit_history")
+  criminalBackground                    String?                       @map("criminal_background")
+  depositMin                            String?                       @map("deposit_min")
+  depositMax                            String?                       @map("deposit_max")
+  depositHelperText                     String?                       @map("deposit_helper_text")
+  disableUnitsAccordion                 Boolean?                      @map("disable_units_accordion")
+  leasingAgentEmail                     String?                       @map("leasing_agent_email")
+  leasingAgentName                      String?                       @map("leasing_agent_name")
+  leasingAgentOfficeHours               String?                       @map("leasing_agent_office_hours")
+  leasingAgentPhone                     String?                       @map("leasing_agent_phone")
+  leasingAgentTitle                     String?                       @map("leasing_agent_title")
+  name                                  String
```

Comment:
Remembering we have a different set of fields that are required for listing publish in core and Detroit, just thinking about the best way for us to manage that

Reply:
good question!

I think that the requirements will need to shared across both the front end and the backend. In the backend DTOs are likely the solution here

but I think we can cross that bridge a bit when we go to bring detroit on to backend 2.0 

you're right though that some of these fields will need to become optional in the db once that happens (that wasn't something I corrected for on my passes through this)